From 8c9cb884ebdc79bb69df13f2bfdf99543233588f Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Mon, 7 Jan 2019 11:32:41 -0500
Subject: [PATCH 03/63] linux-pci-root: remove an unused assignment

scan-build gripes about this, and it's pointless, so it can go.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 src/linux-pci-root.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/linux-pci-root.c b/src/linux-pci-root.c
index a2d9fb04a11..5d1df06119d 100644
--- a/src/linux-pci-root.c
+++ b/src/linux-pci-root.c
@@ -115,7 +115,6 @@ dp_create_pci_root(struct device *dev UNUSED,
                         return new;
                 }
         }
-        off += new;
         sz += new;
 
         debug("returning %zd", sz);
-- 
2.26.2

